lms/djangoapps/courseware/tests/test_tabs.py

```diff
@@ -506,22 +506,22 @@ def setUp(self):
             [],
```

Comment:
I can see that this is an empty list :) - could the comment explain why an empty list is valid?